### PR TITLE
New CLIs

### PR DESCRIPTION
**Pull request by edewata**
The `pki-server ca-cert-request-import` and `pki-server <subsystem>-user-cert-find` commands have been added to support PKI server migration in IPA.

https://github.com/dogtagpki/freeipa/wiki/Migrating-PKI-Server

### CODE REVIEW

File: base/server/src/main/java/org/dogtagpki/server/cli/SubsystemUserCertFindCLI.java [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+            if (certs != null) {
+                boolean first = true;
+                for (X509Certificate cert : certs) {
+
+                    if (first) {
+                        first = false;
+                    } else {
+                        System.out.println();
+                    }
```
Comment (ckelleyRH) [inline base/server/src/main/java/org/dogtagpki/server/cli/SubsystemUserCertFindCLI.java:80]: Can we not do `certs[0]` if we only want the first cert in the array?

File: base/server/python/pki/server/cli/ca.py [hunk 2]
```diff
@@ -731,6 +732,96 @@ def execute(self, argv):
...
+    def __init__(self):
+        super().__init__('import', 'Import certificate request into CA')
+
+    def print_help(self):
+        print('Usage: pki-server ca-cert-request-import [OPTIONS]')
+        print()
+        print('  -i, --instance <instance ID>     Instance ID (default: pki-tomcat)')
+        print('      --csr <path>                 Certificate request path')
+        print('      --format <format>            Certificate request format: PEM (default), DER')
+        print('      --profile <filename>         Bootstrap profile filename')
```
Comment (ckelleyRH) [inline base/server/python/pki/server/cli/ca.py:746]: The help message says profile filename, but the variables (both here and in the related method in `subsystem`) suggest it is the profile ID - I'm not sure which is right (or if they're the same)?

**Review by ckelleyRH — APPROVED** [review]
LGTM, but a couple of questions.

File: base/server/python/pki/server/cli/ca.py (hunk 2 above)
Comment (fmarco76) [reply base/server/python/pki/server/cli/ca.py:746]: AFAIK they are the same but I would prefer to call "profile ID" to be independent from they way profile are stored.

File: base/server/src/main/java/org/dogtagpki/server/cli/SubsystemUserCertFindCLI.java (hunk 1 above)
Comment (fmarco76) [reply base/server/src/main/java/org/dogtagpki/server/cli/SubsystemUserCertFindCLI.java:80]: All certificate are printed! This is just to skip the new line. Is there a problem to move the operation as last one of the for clock and leave for all the certs?

**Review by fmarco76 — APPROVED** [review]
LGTM

File: base/server/python/pki/server/cli/ca.py (hunk 2 above)
Comment (edewata) [reply base/server/python/pki/server/cli/ca.py:746]: Actually, the existing code is using a `profile_id` parameter name for the bootstrap profile, so regardless there will be some sort of "mismatch". Let me merge this as is for now, but we can update it if we figure out a better way to handle this.

**Comment by edewata** [comment]
@ckelleyRH @fmarco76 Thanks!

**Pull request merged**